imageio_ffmpeg/_definitions.py

```diff
@@ -34,6 +36,7 @@ def get_platform():
 
 # Platform string -> ffmpeg filename
 FNAME_PER_PLATFORM = {
+    "osx-arm64": "ffmpeg-osx-arm64-v7", # Apple Silicon
     "osx64": "ffmpeg-osx64-v4.2.2",  # 10.10+
```

Comment:
```suggestion
    "osx64": "ffmpeg-osx-x86-v7.0",  # 10.10+
```

Reply:
Is the `# 10.10+` comment still valid, or does this also need to be updated?

Reply:
It looks like its 10.9 even. (on osxexperts.net it says `MACOSX_DEPLOYMENT_TARGET=10.9`).

Reply:
Ok, I'll change the comment from 10.10+ to 10.9+ instead.